registry/authchallenge_test.go

```diff
@@ -47,14 +47,22 @@ func TestParseChallenge(t *testing.T) {
 				scope:   []string{"repository:chrome:pull"},
 			},
 		},
+		{
+			header:      `Basic realm="https://r.j3ss.co/auth",service="Docker registry"`,
+			errorString: "basic auth required",
+		},
+		{
+			header:      `Basic realm="Registry Realm",service="Docker registry"`,
+			errorString: "basic auth required",
+		},
```

Comment:
any reason for removing these just curious

Reply:
Removing which? These are new test cases :)

Reply:
omg lol long day read the diff wrong sorry